server/db/Database.ts

```diff
@@ -0,0 +1,25 @@
+import { builtinModules } from "module";
```

Comment:
I think this can be removed :).

Reply:
> I think this can be removed :).

That line was a 'quick fix' from VSC, but I changed some names and now it's okay without it.